ncmcm/bundlenet/tests/test_bundlenet_continuous.py

```diff
@@ -1,16 +1,17 @@
+import torch
 import numpy as np
 from ncmcm.bundlenet.utils import prep_data
 from ncmcm.bundlenet.bundlenet import BunDLeNet, train_model
 
 
 def test_bundlenet_training():
     X = np.random.rand(50, 10)
-    B = np.random.random(size=(50,2))
+    B = np.random.random(size=(50,2)).astype(np.float32)
```

Comment:
Why is this necessary?

Reply:
I have made a [small change](https://github.com/NC-MCM/NC-MCM/pull/21/commits/4754ecf1112ec6ae9634948545226707acc41804) so that that’s not necessary anymore